core/src/main/resources/hudson/diagnosis/ReverseProxySetupMonitor/message.jelly

```diff
@@ -36,9 +36,9 @@ THE SOFTWARE.
       }
     );
   </script>
-  <div class="error" id="redirect-error" style="display:none">
+  <div id="redirect-error" class="alert alert-danger" style="display:none">
```

Comment:
This `style` attribute is still here for keeping the current behavior but I would like to revisit this part in midterm.